### DISCUSSION TITLE
The "Status" of block devices fluctuates between 'active' and 'inactive'

### DISCUSSION THREAD

**Discussion by pratmeht** [openebs/openebs#3618] [Q&A]
Hi Openebs Community,

Need a help.

Prior to installing openebs with cstore, we have made a disk available to each K8s Worker nodes. When we do "kubectl get  bd -o wide", we notice all the devices are listed, however their "Status" column shows values 'active' and 'inactive' (which keeps fluctuating at certain interval). Every time, the number of devices which are active or inactive, across all the Worker nodes, keeps altering!
Not sure what that means.

Also, when we use these block devices to create "CStorPoolCluster", it throws an error stating that "block device not active"

```
[root@controlplane30 ~]# view cspc-pratiktest.yaml
[root@controlplane30 ~]# k apply -f cspc-pratiktest.yaml
Error from server (BadRequest): error when creating "cspc-pratiktest.yaml": admission webhook "admission-webhook.cstor.openebs.io" denied the request: invalid cspc specification: invalid pool spec: block device "blockdevice-6d7940d104dac616742b8f394c8672f1" is in not in active state
```

We have 2 queries:
1. On Linux level, when I run "lsblk" command, the device is always visible, then with "kubectl get bd", why does status keep on fluctuating ? 
2. If this fluctuating behaviour is normal, then how do I use them to create "CStorPoolCluster" ?

Regards,
Pratik Mehta

**Comment by avishnu** [comment]
@pratmeht,
What's your K8s environment and what is the OpenEBS version?

**Reply by pratmeht** [reply]
K8s 1.24.2 and Openebs 3.4

**Comment by niladrih** [comment]
Could you fetch the logs from the NDM DaemonSet pod container from the node where the disk is changing states?

**Comment by avishnu** [comment]
Closing as there is no response.
